libraries/streaming-e2e-tests/build/create-react-app.yml

```diff
@@ -0,0 +1,37 @@
+# Create a React app that uses ReactWebChat
+
+steps:
+  - powershell: npx create-react-app react-app
+    displayName: 'create-react-app'
+    workingDirectory: $(RootReactPiecesDir)
+
+  - powershell: npm install botframework-webchat
+    displayName: 'npm install botframework-webchat'
+    workingDirectory: $(RootReactPiecesDir)/react-app
+
+  - powershell: Remove-Item src/App.js, src/index.js, public/index.html
+    displayName: "Delete files we're replacing with custom code in react app"
+    workingDirectory: $(RootReactPiecesDir)/react-app
+
+  - powershell: New-Item .env
+    displayName: 'Create .env in React app'
+    workingDirectory: $(RootReactPiecesDir)/react-app
+
+  - powershell: |
+      copy ../custom-app-code/App.js, ../custom-app-code/index.js, ../custom-app-code/.gitignore ./src/
+      copy ../custom-app-code/index.html ./public
+    displayName: "Copy custom files into react-app"
+    workingDirectory: $(RootReactPiecesDir)/react-app
+
+  - powershell: |
+      Add-Content -Path "./.env" -Value BOT_HOSTNAME=$(BotName)
+
+      Get-Content -Path "./.env"
+    displayName: "Write bot's website host name to React app's .env"
+    workingDirectory: $(RootReactPiecesDir)/react-app
+
```

Comment:
Run `npm run build` here.

Then pack the `/build/**/*` and publish it as artifact.